src/sentry/migrations/0099_fix_project_platforms.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-09-02 21:33
+from __future__ import unicode_literals
+
+import logging
+
+from django.db import migrations
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def fix_project_platform(apps, schema_editor):
+    """
+    Find projects whose platform is either 'python-tracing' or 'node-tracing',
+    and change them to be either 'python' or 'node' respectively.
+    """
+    Project = apps.get_model("sentry", "Project")
+
+    for project in RangeQuerySetWrapperWithProgressBar(Project.objects.all()):
```

Comment:
This is going to scan all the projects in production. You could apply conditions to this query to cut down the amount of rows you look at by doing `.filter(platform__in=["node-tracing", "python-tracing"])`

Reply:
@wedamija can you confirm?

Reply:
Usually when I've done this kind of thing @mattrobenolt's advice here is to not do this and iterate through all rows. There's no index on this field, so it means that we'd be constantly table scanning to fetch the results.

Reply:
The docs here seems to match what @wedamija mentioned; and it was a pattern I chose to use https://develop.sentry.dev/database-migrations/#filters